libraries/redcore/layouts/toolbar/button/versions.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * @package     Redcore
+ * @subpackage  Layout
+ *
+ * @copyright   Copyright (C) 2008 - 2015 redCOMPONENT.com. All rights reserved.
+ * @license     GNU General Public License version 2 or later, see LICENSE.
+ */
+
+defined('JPATH_REDCORE') or die;
+
+$height    = $displayData['height'];
+$width     = $displayData['width'];
+$itemId    = (int) $displayData['itemId'];
+$typeId    = (int) $displayData['typeId'];
+$typeAlias = $displayData['typeAlias'];
+$title     = $displayData['title'];
+
+$link = 'index.php?option=com_contenthistory&amp;view=history&amp;layout=modal&amp;tmpl=component&amp;item_id=' . $itemId .
+	'&amp;type_id=' . $typeId . '&amp;type_alias=' . $typeAlias . '&amp;' . JSession::getFormToken() . '=1';
+?>
+
+<a rel="{handler: 'iframe', size: {x: <?php echo $height ?>, y: <?php echo $width ?>}}"
+	href="<?php echo $link ?>"
+	title="<?php echo $title ?>"
+	class="btn btn-default modal_jform_contenthistory">
+	<i class="icon-archive"></i> <?php echo $displayData['title']; ?>
```

Comment:
I have a thought about this, that I don't have it very clear in my mind. 

And you may need to use different icons depending on the Font you have loaded. Then, maybe for redCORE that uses the FontAwesome you will call the **icon-archive** while you can have a joomla 3 Bootstrap 2.3 IcoMoon font and you can call it **icon-othername**

So it will be task for the layout to define the icon?